site/_posts/2021-09-15-rbvmomi-maintenance.md

```diff
@@ -0,0 +1,25 @@
+---
+title: RbVmomi Maintenance
+date: 2021-09-21 18:00 UTC
```

Comment:
This needs to be updated once we actually get to posting it.

Reply:
Incidentally, you don't need the time...the date is sufficient.

Reply:
That's interesting, a lot of posts have the time :shrug e.g. `site/_posts/2021-04-26-Announcing-Lasker-Beta.md:date: 2021-04-26 13:27:43 UTC`

Updated date and removed time